flexget/api/core/cached.py

```diff
@@ -22,15 +23,15 @@ class CachedResource(APIResource):
     @api.response(BadRequest)
     @api.response(APIError)
     @api.doc(parser=cached_parser)
-    def get(self, session=None):
+    def get(self, session: None = None) -> Response:
```

Comment:
type shouldn't be None, it's Session

Reply:
`session` gets injected by the `@with_session` decorator, which is not applied to this method.